tests/Processors/ExpandEnumsTest.php

```diff
@@ -65,11 +65,12 @@ public function testExpandBackedStringEnum(): void
         self::assertEquals(['draft', 'published', 'archived'], $schema->enum);
     }
 
-    /**
-     * @requires PHP >= 8.1
-     */
     public function expandEnumClassStringFixtures(): iterable
     {
+        if (!class_exists('\\ReflectionEnum')) {
```

Comment:
Would PHP run this even if the test is disabled :/

Reply:
It may be a specification of dataProvider, but it seems to be working.
https://github.com/k2tzumi/swagger-php/runs/8181815888?check_suite_focus=true#step:5:36